tests/network/sriov.go

```diff
@@ -806,3 +809,46 @@ func trimRawString2JSON(input string) string {
 	}
 	return input[startIdx : endIdx+1]
 }
+
+func getRunningVMIDomainSpec(vmi *v1.VirtualMachineInstance) (*launcherApi.DomainSpec, error) {
+	runningVMISpec := launcherApi.DomainSpec{}
+	cli := kubevirt.Client()
+
+	domXML, err := getRunningVirtualMachineInstanceDomainXML(cli, vmi)
+	if err != nil {
+		return nil, err
+	}
+
+	err = xml.Unmarshal([]byte(domXML), &runningVMISpec)
+	return &runningVMISpec, err
+}
+
+func getRunningVirtualMachineInstanceDomainXML(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) (string, error) {
+	// get current vmi
+	freshVMI, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(context.Background(), vmi.Name, metav1.GetOptions{})
+	if err != nil {
+		return "", fmt.Errorf("failed to get vmi, %s", err)
+	}
+
+	vmiPod, err := libpod.GetPodByVirtualMachineInstance(freshVMI, freshVMI.Namespace)
+	if err != nil {
+		return "", err
+	}
+
+	command := []string{"virsh"}
+	if kutil.IsNonRootVMI(freshVMI) {
```

Comment:
NonRoot feature is GA, meaning all VMs are non-root, I thin we can remove this check.
@orelmisan please correct me if I am wrong.

Reply:
The non root mode is indeed GA and is the default.
One has to explicitly set a FG on KubeVirt CR in order for the VM to be rootful.
Rootful VMs are about to be removed from KubeVirt so I think we could remove this condition (in a dedicated commit with proper reasoning).